docs/src/CP/performance_test.md

```diff
@@ -0,0 +1,31 @@
+# Performance test - Gecode vs SeaPearl
+
+The objective is to compare the execution time between SeaPearl and a commercial CP Solver.
+
+We expect SeaPearl to be less efficient for 2 reasons:
+* Information is collected on each node in order to do further analysis (RL).
+* Some constraints algorithms are not state-of-the-art algorithms.
+
+## Method
+
+The commercial CP solver used is Gecode and the model has been implemented using MiniZinc. 
+
+The problem used for the experience is the Kidney Exchange Problem (KEP) using 7 instances from size 5 to 35 (step size of 5). 
+
+A basic model has been implemented in MiniZinc and SeaPearl using the same logic (variables, constraints, heuristics...).
+
+## Results and conclusions
+
+The main observations from the experience are:
+
+* The time of execution grows over-polynomially with the size of the instance. This result was expected as the KEP is a combinatorial problem.
```

Comment:
In fact, the over-polynomial behavior is not specifically a characteristic of combinatorial problems even if in practice, most of the problems suffers from a exponential time complexity for solving w.r.t input size.

Reply:
All right, I have modified this sentence to be more precise